static/js/compose_error.js

```diff
@@ -0,0 +1,32 @@
+import $ from "jquery";
+
+import * as common from "./common";
+
+export function compose_error(error_html, bad_input) {
+    $("#compose-send-status")
+        .removeClass(common.status_classes)
+        .addClass("alert-error")
+        .stop(true)
+        .fadeTo(0, 1);
+    $("#compose-error-msg").html(error_html);
+    $("#compose-send-button").prop("disabled", false);
+    $("#sending-indicator").hide();
+    if (bad_input !== undefined) {
+        bad_input.trigger("focus").trigger("select");
+    }
+}
+
+export function compose_not_subscribed_error(error_html, bad_input) {
+    $("#compose-send-status")
+        .removeClass(common.status_classes)
+        .addClass("home-error-bar")
```

Comment:
I'd call this `compose_error.show_not_subscribed(...)`.

That said, is the only difference here the `home-error-bar` class?  If so, we should think about a refactoring to let us pass an `error_class` parameter as well once this is merged.

Reply:
I also thought the same first. But it also does another task, so right now didn't changed it. I will see if that will be possible and change it.